packages/design-system/src/components/contextMenu/components/separator.js

```diff
@@ -25,10 +25,19 @@ import { useContextMenu } from '../contextMenuProvider';
 
 const Line = styled.div`
   display: block;
-  height: 1px;
   background-color: ${({ theme }) => theme.colors.divider.primary};
-  margin: 8px auto;
-  width: ${({ $isIconMenu }) => ($isIconMenu ? 40 : 100)}%;
+  ${({ $isHorizontal, $isIconMenu }) =>
+    $isHorizontal
+      ? `
+        width: 1px;
+        height: 86%; /* 31px out of 36px */
```

Comment:
What's the 31px? It's just a value from Figma?

Reply:
Yes, I can change it to 32px, which is probably the intent, yes.